site/src/pages/UsersPage/UsersPageView.stories.tsx

```diff
@@ -22,29 +34,23 @@ const Template: Story<UsersPageViewProps> = (args) => (
 )
 
 export const Admin = Template.bind({})
-Admin.args = {
-  users: [MockUser, MockUser2],
-  roles: MockAssignableSiteRoles,
-  canEditUsers: true,
-}
 
 export const SmallViewport = Template.bind({})
-SmallViewport.args = {
-  ...Admin.args,
-}
 SmallViewport.parameters = {
   chromatic: { viewports: [600] },
 }
 
 export const Member = Template.bind({})
-Member.args = { ...Admin.args, canEditUsers: false }
+Member.args = { canEditUsers: false }
 
 export const Empty = Template.bind({})
-Empty.args = { ...Admin.args, users: [] }
+Empty.args = { users: [] }
```

Comment:
I think not having `isNonInitialPage` here is causing `Cond` to error with `A non-final Cond in a ChooseOne does not have a condition prop.`.